README.md

```diff
@@ -51,6 +51,29 @@ KubeEdge is composed of the following components:
 
 <img src="./docs/images/kubeedge_arch.png">
 
+## Compatibility matrix
+
+### Kubernetes compatibility
+
+|                     | Kubernetes 1.10 | Kubernetes 1.11 | Kubernetes 1.12 | Kubernetes 1.13 | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 |
+|---------------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------| --------------- |
+| KubeEdge 1.0        | ✓               | -               | -               | -               | -               | -               | -               |
+| KubeEdge 1.1        | +               | +               | +               | +               | ✓               | ✓              | -               |
+| KubeEdge HEAD       | +               | +               | +               | +               | ✓               | ✓              | -               |
+
+Key:
+* `✓` KubeEdge and the Kubernetes version are exactly compatible.
+* `+` KubeEdge has features or API objects that may not be present in the Kubernetes version.
+* `-` The Kubernetes version has features or API objects that KubeEdge can't use.
+
+### Golang dependency
+
+|                     | Golang 1.10    | Golang 1.11     | Golang 1.12     | Golang 1.13     |
```

Comment:
Do we have plan to upgrade golang version to 1.13? 1.13 is formally released.

Reply:
Yes we will, can upgrade the k8s to 1.16